ember-bem-helpers/package.json

```diff
@@ -0,0 +1,75 @@
+{
+  "name": "ember-bem-helpers",
+  "version": "0.4.0",
+  "description": "BEM helpers for Ember.js applications",
+  "keywords": [
+    "ember-addon",
+    "BEM",
+    "helper"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/retailnext/ember-bem-helpers.git"
+  },
+  "license": "MIT",
+  "author": "RetailNext",
+  "exports": {
+    ".": "./dist/index.js",
+    "./*": "./dist/*.js",
+    "./addon-main.js": "./addon-main.cjs"
+  },
+  "files": [
+    "addon-main.cjs",
+    "src/**/*.js",
+    "declarations",
+    "dist"
+  ],
+  "scripts": {
+    "build": "rollup --config",
+    "lint": "concurrently 'npm:lint:*(!fix)' --names 'lint:'",
+    "lint:fix": "concurrently 'npm:lint:*:fix' --names 'fix:'",
+    "lint:hbs": "ember-template-lint . --no-error-on-unmatched-pattern",
+    "lint:hbs:fix": "ember-template-lint . --fix --no-error-on-unmatched-pattern",
+    "lint:js": "eslint . --cache",
+    "lint:js:fix": "eslint . --fix",
+    "prepack": "rollup --config",
+    "start": "rollup --config --watch",
+    "test": "echo 'A v2 addon does not have tests, run tests in test-app'"
+  },
+  "dependencies": {
+    "@embroider/addon-shim": "1.8.7",
+    "decorator-transforms": "1.0.3"
```

Comment:
Dependencies shouldn't be pinned. Consumers can have newer versions.

Reply:
Done